### ISSUE TITLE
Disable Automatic Hiding of Reported Content When Deleted

### ISSUE THREAD

**Issue by thebluecorsair** [Retrospring/bugs#31]
At the current time, whenever any form of action is taken on reported content and the content was deleted, the associated report is also hidden and removed from the moderation panel.

This is counter-productive as it does not allow other moderators to see what happened to the content as well as the comments or votes given to it. I feel that this should be adjusted to continue to show the report even if it's related content was removed and then manually require it's deletion.

**Comment by yukimono** [comment]
That's mostly because reports are polymorphic and don't hold a lot of unique data themselves, the actual reported item is an id number which then gets turned into the actual comment, question or answer.

tl;dr the report has no data on what the reported thing actually is, when the item is deleted, the report cannot resolve the item's data anymore, to change this we'd need to modify how we're deleting content (i.e. don't delete anything at all and just mark the item as "offline", similar to how notifications and inboxed questions are marked as "read")
